### PR TITLE
Add instructions for log downloads

### PR DESCRIPTION
**Pull request by jaxxzer**

### CODE REVIEW

**Comment by jaxxzer** [comment]
'Other Parameters' are already configured correctly by default.

**Comment by jaxxzer** [comment]
It also looks like QGC has moved the log download to a button, so I will need to update that.

**Comment by rjehangir** [comment]
Okay, let me know when it's ready and I'll merge.

**Comment by rjehangir** [comment]
@jaxxzer - Have you tried this with a more recent daily build of QGC? Don added a log download page that makes this a little easier, I believe. It shows up next to the flight tab.

File: pages/troubleshooting.md [hunk 1]
```diff
@@ -48,17 +48,15 @@ DataFlash logs can be retrieved in two ways:
 
 	- Log downloading via QGroundControl
 
-		Click the 'Widgets' icon in the top menu bar, and click the 'Log Download' option.
-
-		<img src="/images/log-download.png" class="img-responsive img-center" style="max-height:400px;">
+		Click the Analyze icon at the top of the window. The icon looks like a magnifying glass over a document.
```
Comment (jaxxzer) [inline pages/troubleshooting.md:51]: This was updated to use the icon in an updated version of QGC.

**Pull request merged**